SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -2253,7 +2253,7 @@ unsigned long CEulerSolver::SetPrimitive_Variables(CSolver **solver_container, c
   SU2_OMP_FOR_STAT(omp_chunk_size)
   for (unsigned long iPoint = 0; iPoint < nPoint; iPoint ++) {
 
-    /*--- Compressible flow, primitive variables nDim+5, (T, vx, vy, vz, P, rho, h, c, lamMu, eddyMu, ThCond, Cp) ---*/
+    /*--- Compressible flow, primitive variables nDim+9, (T, vx, vy, vz, P, rho, h, c, lamMu, eddyMu, ThCond, Cp) ---*/
```

Comment:
Just a thought: It would be nice to switch `P` and `T` for the comp-var to be consistent with the inc-variable. Not sure how easy that is to change though (maybe just change the `GetPressure` and `GetTemperature` call and one is good).  But maybe this is also a stupid idea.

Reply:
Even nicer if the incompressible solvers would just use the solution directly instead of storing P u v w T twice.
Changing storage places would break all the numerics, as they get a pointer to the start of the primitives.